examples/jenkins-master/jenkins-slave-template.json

```diff
@@ -44,7 +50,7 @@
         "name": "${IMAGE_STREAM_NAME}"
       },
       "spec": {
-        "dockerImageRepository": "openshift/${IMAGE_STREAM_NAME}"
+        "dockerImageRepository": "${NAMESPACE}/${IMAGE_STREAM_NAME}"
```

Comment:
this is a docker pullspec, not an imagestream, so specifying the namespace is the wrong thing to do.

This template is a bit complicated in what it needs to do, it's working with both docker images as well as imagestreams, and sharing a single parameter for both.  We can talk about how to address it tomorrow.